docs/en/ingest-management/elastic-agent/configuration/elastic-agent-standalone-features.asciidoc

```diff
@@ -0,0 +1,38 @@
+[[elastic-agent-standalone-feature-flags]]
+= Configure feature flags for standalone {agent}s
+
+++++
+<titleabbrev>Feature flags</titleabbrev>
+++++
+
+include::{fleet-repo-dir}/standalone-note.asciidoc[]
+
+The Feature Flags section of the `elastic-agent.yml` config file contains settings in {agent} that are disabled by default. Having these settings behind a feature flag prevents accidentally exposing any breaking changes that might be inconsistent with the naming formats expected in your configured {agent} output.
+
+To enable any of the settings listed on this page, change the associated `enabled` flag from `false` to `true`.
+
+[source,yaml]
+----
+agent.features:
+  mysetting:
+    enabled: true
+----
+
+[discrete]
+[[elastic-agent-standalone-feature-flag-settings]]
+== Feature flag configuration settings
+
+You can specify the following settings in the Feature Flag section of the
+`elastic-agent.yml` config file.
+
+Fully qualified domain name (FQDN)::
+When enabled, information provided about the current host through the <<host-provider,host.name>> key, in events produced by {agent}, is in FQDN format (`somehost.example.com` rather than `somehost`). This helps you to distinguish between hosts on different domains that have similar names. With `fqdn` enabled, the fully qualified hostname allows each host to be more easily identified when viewed in {kib}.
++
+To enable fully qualified domain names set `enabled: true` for the `fqdn` setting:
++
+["source","yaml",subs="attributes"]
+----
+agent.features:
+  fqdn:
+    enabled: true
+----
```

Comment:
I think somewhere within this section on FQDN, we need to address https://github.com/elastic/ingest-dev/issues/1641#issuecomment-1482574954, specifically the bit about pre-requisites for FQDN reporting to work as expected after users enable the FQDN feature flag.  

For FQDN reporting to work as expected, the hostname of the current host must either:
- have a CNAME entry defined in DNS, or
- have one of its corresponding IP addresses respond successfully to a reverse DNS lookup.

If neither of these pre-requisites are satisfied, `host.name` will continue to report the hostname of the current host as if the FQDN feature flag were not enabled.

Reply:
Thanks for catching that! I've added the prereqs, as shown:

![Screenshot 2023-04-03 at 11 29 02 AM](https://user-images.githubusercontent.com/41695641/229556707-e94522ca-4a09-4017-b784-1474f240a87d.png)

Reply:
> If neither of these pre-requisites are satisfied, `host.name` will continue to report the hostname of the current host as if the FQDN feature flag were not enabled.

Do you think it's worth mentioning this "fallback" behavior as well or would that be redundant information for users?  It's certainly not _actionable_ information for users but it might explain why they're still seeing the host's hostname despite enabling the FQDN feature flag.  WDYT?

Reply:
@kilfoyle Looks like we crossed wires there 🙂.  I see you already incorporated my last comment in the doc.  Thanks!

Reply:
I missed that part by accident on the first round. 😄